Assets/_Application/DataTypeAdapters/Tiles3DImportAdapter.cs

```diff
@@ -0,0 +1,69 @@
+using UnityEngine;
+using UnityEngine.Events;
+using Netherlands3D.DataTypeAdapters;
+using Netherlands3D.Functionalities.OGC3DTiles;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace Netherlands3D.Twin.DataTypeAdapters
+{
+    [CreateAssetMenu(menuName = "Netherlands3D/Adapters/Tiles3DImportAdapter", fileName = "Tiles3DImportAdapter", order = 0)]
+    public class Tiles3DImportAdapter : ScriptableObject, IDataTypeAdapter
+    {
+        [SerializeField] private Tile3DLayerGameObject layerPrefab;
+        [SerializeField] private UnityEvent<string> displayErrorMessageEvent;
+
+        public void Execute(LocalFile localFile)
+        {
+            var layerParent = GameObject.FindWithTag("3DTileParent").transform;
```

Comment:
TagNaam, of zelfs een GameObject naam, kan beter een serialized field zijn om de ScriptableObject - als we iets wijzigen aan de structuur van de applicatie dan kan in de editor deze SO ook worden aangepast - en het maakt het duidelijker dit dit gedaan wordt omdat je een soort van herinnering hebt in de editor of boven in de class